eva-accession-release/src/test/java/uk/ac/ebi/eva/accession/release/io/ContigMongoReaderTest.java

```diff
@@ -0,0 +1,60 @@
+package uk.ac.ebi.eva.accession.release.io;
+
+import com.lordofthejars.nosqlunit.annotation.UsingDataSet;
+import com.lordofthejars.nosqlunit.mongodb.MongoDbConfigurationBuilder;
+import com.lordofthejars.nosqlunit.mongodb.MongoDbRule;
+import com.mongodb.MongoClient;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.batch.item.ExecutionContext;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.ApplicationContext;
+import org.springframework.data.mongodb.core.MongoTemplate;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import uk.ac.ebi.eva.accession.core.configuration.MongoConfiguration;
+import uk.ac.ebi.eva.accession.release.test.configuration.MongoTestConfiguration;
+import uk.ac.ebi.eva.accession.release.test.rule.FixSpringMongoDbRule;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+@RunWith(SpringRunner.class)
+@TestPropertySource("classpath:application.properties")
+@UsingDataSet(locations = {
+        "/test-data/dbsnpClusteredVariantEntity.json"})
+@ContextConfiguration(classes = {MongoConfiguration.class, MongoTestConfiguration.class})
+public class ContigMongoReaderTest {
+
+    private static final String TEST_DB = "test-db";
+
+    private static final String ASSEMBLY_ACCESSION = "GCF_000409795.2";
+
+    @Autowired
+    private MongoClient mongoClient;
+
+    //Required by nosql-unit
+    @Autowired
+    private ApplicationContext applicationContext;
+
+    @Rule
+    public MongoDbRule mongoDbRule = new FixSpringMongoDbRule(
+            MongoDbConfigurationBuilder.mongoDb().databaseName(TEST_DB).build());
+
+    @Test
+    public void basicRead() {
+        ContigMongoReader reader = new ContigMongoReader(ASSEMBLY_ACCESSION, mongoClient, TEST_DB);
+        reader.open(new ExecutionContext());
+        String contig;
+        List<String> contigs = new ArrayList<>();
+        while ((contig = reader.read()) != null) {
+            contigs.add(contig);
+        }
+        assertEquals(2, contigs.size());
```

Comment:
Since they are not many, we could check their values too to ensure that the same one is not duplicated or similar.